src/load/azext_load/data_plane/utils/argtypes.py

```diff
@@ -523,3 +523,67 @@
     choices=utils.get_enum_values(TriggerState),
     help="List all the schedules in the resource which are in the provided states.",
 )
+
+notification_rule_test_ids = CLIArgumentType(
+    options_list=["--test-ids"],
+    nargs="+",
+    validator=validators.validate_notification_rule_test_ids,
+    help="Test ids for notification rule.",
+)
+
+notification_display_name = CLIArgumentType(
+    option_list=["--display-name"],
+    type=str,
+    help="Display name of notification rule.",
+)
+
+notification_rule_id = CLIArgumentType(
+    validator=validators.validate_notification_rule_id,
+    option_list=["--notification-rule-id"],
+    type=str,
+    help="Identifier for the notification rule.",
+)
+
+action_groups = CLIArgumentType(
+    option_list=["--action-groups"],
+    nargs="*",
+    help="Action group for notification rule.",
+)
+
+notification_rule_event = CLIArgumentType(
+    option_list=["--event"],
+    validator=validators.validate_event,
+    nargs="+",
+    action="append",
+    help="Event for notification rule.",
+)
+
+notification_rule_remove_event = CLIArgumentType(
+    option_list=["--remove-event"],
+    nargs="+",
+    validator=validators.validate_remove_event,
+    action="append",
+    help="Event to be removed for notification rule.",
```

Comment:
Provide the event id of the event to be removed from the notification rule. Format should be --remove-event event-id=event1.